spec/controllers/curation_concern/permissions_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe CurationConcern::PermissionsController do
+  let(:user) { FactoryGirl.create(:user) }
+  before { sign_in user }
+
+  describe "#confirm" do
+    let(:generic_work) { FactoryGirl.create(:generic_work, user: user) }
+
+    it "should draw the page" do
```

Comment:
@scherztc I think you need to add render_views before the page will get drawn: https://www.relishapp.com/rspec/rspec-rails/v/2-1/docs/controller-specs/render-views